.github/workflows/link-pr-to-source-issue.yml

```
@@ -0,0 +1,123 @@
+          if [ -z "$PREPEND_LINES" ]; then
+            echo "PR body already contains required closing references; skipping."
+            exit 0
+          fi
+
+          NEW_BODY=$(printf '%b\n\n%s' "$PREPEND_LINES" "$CURRENT_BODY")
+
+          gh api \
+            "repos/${REPO}/pulls/${PR_NUMBER}" \
+            -X PATCH \
+            -f body="$NEW_BODY"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)